hbase-common/src/main/java/org/apache/hadoop/hbase/util/UnsafeAvailChecker.java

```diff
@@ -33,12 +35,16 @@
   private static final Logger LOG = LoggerFactory.getLogger(UnsafeAvailChecker.class);
   private static boolean avail = false;
   private static boolean unaligned = false;
+  // Flag that can be set by clients to enable/disable Unsafe package usage
+  private static boolean useUnsafe;
 
   static {
     avail = AccessController.doPrivileged(new PrivilegedAction<Boolean>() {
       @Override
       public Boolean run() {
         try {
+          final Configuration conf = HBaseConfiguration.create();
+          useUnsafe = conf.getBoolean("hbase.unsafe.usage.enabled", true);
```

Comment:
Make this configuration key a string literal, please.

Reply:
Done @apurtell